tap_zoom/schemas/users.json

```diff
@@ -102,6 +114,56 @@
         "null",
         "integer"
       ]
+    },
+    "user_created_at": {
```

Comment:
on line 70 we have `created_at` which seems to be a deprecated field [doc](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/users)